src/fitsviewer.py

```diff
@@ -23,25 +24,24 @@
 from __future__ import unicode_literals
 
 import sip
+from pathlib import Path
+from functools import partial
 API_NAMES = [
     'QDate', 'QDateTime', 'QString', 'QTextStream',
     'QTime', 'QUrl', 'QVariant'
     ]
 API_VERSION = 2
 for name in API_NAMES:
     sip.setapi(name, API_VERSION)
-
-from PyQt4 import QtCore, QtGui
-from PyQt4.QtCore import Qt
-
 import os
 import sys
-if sys.version_info >= (3, 0):
-    from mainwindow_form3 import Ui_MainWindow
-    from plotwindow_form3 import Ui_Dialog
-else:
-    from mainwindow_form import Ui_MainWindow
-    from plotwindow_form import Ui_Dialog
+
+from PyQt5 import QtWidgets, QtGui, QtCore
+from PyQt5.QtCore import Qt
+
+from src.mainwindow_form3 import Ui_MainWindow
```

Comment:
Maybe drop the "3" prefix, since Python 2 support is gone? I think with Python 2 being over 10 years old, it's just confusing what it is about.